### PR TITLE
Meka | Dog | Drake | Sitting borgs have a chance to crush human skulls by sitting on them.

### PR DESCRIPTION
**Pull request by nevimer**
## About The Pull Request

This PR will allow any sitting borg type to crush someones skull with their module. This spawned out of a joke. 

https://user-images.githubusercontent.com/77420409/209723307-bddc172d-cb1c-4658-b503-aff670b72c52.mp4

https://user-images.githubusercontent.com/77420409/209723455-43e8b42e-02ea-4fab-936f-e86f0d2036ed.mp4

## How This Contributes To The Skyrat Roleplay Experience

It could be funny sometimes, I guess!

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

  https://user-images.githubusercontent.com/77420409/209723307-bddc172d-cb1c-4658-b503-aff670b72c52.mp4
</details>

## Changelog

:cl:
add: sitting enabled borgs can now crush peoples faces (accidentally or intentionally)
/:cl:

### CODE REVIEW

**Comment by HoodMilk** [comment]
what the fuck

File: modular_skyrat/master_files/code/modules/mob/living/sillicon/robot.dm [hunk 1]
```diff
@@ -64,7 +64,15 @@
 
 /mob/living/silicon/robot/robot_lay_down()
 	. = ..()
-
+	for(var/mob/living/carbon/human/human in orange(1))
```
Comment (Zonespace27) [inline modular_skyrat/master_files/code/modules/mob/living/sillicon/robot.dm]: just do `in loc`

File: modular_skyrat/master_files/code/modules/mob/living/sillicon/robot.dm [hunk 2]
```diff
@@ -64,7 +64,15 @@
...
+		if(human && prob(1))
```
Comment (Zonespace27) [inline modular_skyrat/master_files/code/modules/mob/living/sillicon/robot.dm]: why prob 1, also the first part of the check will never not be true

File: modular_skyrat/master_files/code/modules/mob/living/sillicon/robot.dm [hunk 3]
```diff
@@ -64,7 +64,15 @@
...
+			. = human.apply_damage(125, BRUTE, BODY_ZONE_HEAD, wound_bonus = 50)
+			if(.)
+				human.visible_message(
+					"[name] slams their PHAT BOOTY into [human]'s head!",
```
Comment (Zonespace27) [inline modular_skyrat/master_files/code/modules/mob/living/sillicon/robot.dm:72]: :suspect:

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

**Comment by dolphinemote** [comment]
holy based

**Comment by Stalkeros2** [comment]
░░░░░░░░░░░░▄▄                     
░░░░░░░░░░░█░░█                  
░░░░░░░░░░░█░░█                  
░░░░░░░░░░█░░░█                  
░░░░░░░░░█░░░░█                  
███████▄▄█░░░░░██████▄ 
▓▓▓▓▓▓█░░░░░░░░░░░░░░█ 
▓▓▓▓▓▓█░░░░ Valve, ░░░░░█ 
 ▓▓▓▓▓▓█░░░░Add This░░░░█  
▓▓▓▓▓▓█░░░░Please!░░░░░█ 
▓▓▓▓▓▓█░░░░░░░░░░░░░░█ 
▓▓▓▓▓▓█████░░░░░░░░░█   
██████▀░░░░▀▀██████▀

**Comment by Paxilmaniac** [comment]
This is the pr we really need

**Comment by StrangeWeirdKitten** [comment]
Unironic borg buff. Ironic borg erp nerf.

**Comment by tf-4** [comment]
should gib

**Comment by Nerev4r** [comment]
can-we-ban-this-guy

**Pull request closed without merging**